### PR TITLE
[11.0][IMP] Add archived field to calendar

### PR DESCRIPTION
**Pull request by jarroyomorales**
Fast-track

### CODE REVIEW

**Comment by jarroyomorales** [comment]
@etobella Please review, reusing this PR to be able to archive calendars

**Comment by jarroyomorales** [comment]
@etobella Aprovecho para mejorar unas traducciones del cb_hr_views.
Cuando quieras se puede mergear esto

File: cb_hr_views/views/resource_calendar_views.xml [hunk 1]
```diff
@@ -27,16 +27,25 @@
             <field name="company_id" position="attributes">
                 <attribute name="invisible">1</attribute>
             </field>
-            <div name="button_box" position="attributes">
+            <xpath expr="//div[@name='button_box']/button[1]" position="attributes">
```
Comment (etobella) [inline cb_hr_views/views/resource_calendar_views.xml]: Change this please...

**Review by etobella — CHANGES_REQUESTED** [review]

**Comment by jarroyomorales** [comment]
@etobella  Comment attended

**Review by etobella — APPROVED** [review]

**Pull request merged**